published-namespaces.txt

```diff
@@ -108,10 +108,81 @@ AndroidX.Car.Widget
 AndroidX.Car.Widget.ItemDecorators
 AndroidX.CardView.Widget
 AndroidX.Collection
+AndroidX.Compose.Animation
 AndroidX.Compose.Animation.Graphics
 AndroidX.Compose.Animation.Graphics.Res
 AndroidX.Compose.Animation.Graphics.Vector
 AndroidX.Compose.Animation.Graphics.Vector.Compat
+AndroidX.Compose.Animation.Internal
+AndroidX.Compose.Foundation.Layout
+AndroidX.Compose.Material
+AndroidX.Compose.Material.Icons
+AndroidX.Compose.Material.Icons.Filled
+AndroidX.Compose.Material.Icons.Outlined
+AndroidX.Compose.Material.Icons.Rounded
+AndroidX.Compose.Material.Icons.Sharp
+AndroidX.Compose.Material.Icons.TwoTone
+AndroidX.Compose.Material.Internal
+AndroidX.Compose.Material.Pullrefresh
```

Comment:
The following compound words need capitalization fixes:
PullRefresh
HapticFeedback
CoreShims
DataSource
ViewInterop
MaterialCore
ProtoLayout

Reply:
good spot. I opened file, but did not finish it.